cloud_service/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -27,39 +27,76 @@ The following sections provide an overview of installation steps and links to th
 [id="securing-rh-cloud-clusters"]
 === Securing {osp} clusters
 
-To secure {osp} clusters by using the Operator, perform the following steps:
+You can secure {osp} clusters by using the Operator, Helm charts, or the `roxctl` CLI.
+
+[id="overview-installing-cloud-secured-clusters-osp-operator"]
+==== Securing {osp} clusters by using the Operator
 
 . Verify that the clusters you want to secure meet the xref:../cloud_service/acscs-default-requirements.adoc#acscs-default-requirements[requirements].
 . In the {cloud-console}, xref:../cloud_service/installing_cloud_ocp/cloud-create-instance-ocp.adoc#cloud-create-instance-ocp[create an *ACS Instance*].
 . On each {osp} cluster you want to secure, xref:../cloud_service/installing_cloud_ocp/cloud-ocp-create-project.adoc#cloud-ocp-create-project[create a project named `stackrox`]. This project will contain the resources for {product-title-managed-short} secured clusters.
-. In the ACS Console, xref:../cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-generate.adoc#init-bundle-cloud-ocp-generate[create an init bundle]. The init bundle contains secrets that allow communication between {product-title-managed-short} secured clusters and the ACS Console.
-. On each {osp} cluster, xref:../cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-apply#init-bundle-cloud-ocp-apply[apply the init bundle] by using it to create resources.
+. Create the mechanism that the Central instance, also called Central, will use to set up communication with the secured clusters. You will either create an init bundle or a cluster registration secret (CRS). Do one of these actions:
+* In the ACS Console, xref:../cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-generate.adoc#init-bundle-cloud-ocp-generate[generate an init bundle]. The init bundle contains secrets that allow communication between {product-title-managed-short} secured clusters and Central.
+* Log in to Central and use the `roxctl` CLI to xref:../cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-generate.adoc#roxctl-generate-init-bundle_init-bundle-cloud-ocp-generate[generate an init bundle].
+* Log in to Central and use the `roxctl` CLI to xref:../cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-generate.adoc#crs-generate-roxctl_init-bundle-cloud-ocp-generate[generate a CRS].
```

Comment:
We are already logged into Central, right? 
```suggestion
* In Central, use the `roxctl` CLI to xref:../cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-generate.adoc#crs-generate-roxctl_init-bundle-cloud-ocp-generate[generate a CRS].
```

Reply:
I saw that you have two steps that start "Log into Central." After you log in the first time, do you have to do it again?

Reply:
No, you are doing one of the 3 bullets - 1) Use the ACS Console and create the init bundle; 2) log in to central and create the init bundle; OR 3) log in to central and create the CRS.

If you pick bullet 1, you use the ACS Console (GUI). If you pick bullet 2 or bullet 3, those each require logging in to Central.  In the previous steps, you are in the Cloud Console (not Central).